### ISSUE TITLE
[Bug] Consider increasing contrast ratio of drop-down symbol in Collections (for Dark Theme)

### ISSUE THREAD

**Issue by neha-b2001** [mozilla-mobile/fenix#19307]
[comment]: # (Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean)
[comment]: # (Every issue should have the exact bug and steps to reproduce described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
[comment]: # (Read https://github.com/mozilla-mobile/fenix#i-want-to-file-an-issue for more information)

## Steps to reproduce
1. Install and open Google Accessibility Scanner.
2. Open Firefox Nightly/Preview. Ensure that the theme is **Dark**. 
3. Select 1 or more tabs (from tab list or grid view) and add them to a collection (i.e. create a collection).
4. Come back to main screen of browser. Under the '**Collections**' heading, you can see the name of your collection and a small drop-down symbol beside it. 

![image](https://user-images.githubusercontent.com/67039214/116566184-2a634e00-a924-11eb-9c93-d6bfc4c8f63e.png)

### Expected behavior
No suggestions by Google Accessibility Scanner.
### Actual behavior
A suggestion is made to **increase text contrast ratio** of this symbol (in Dark theme, this drop-down symbol is not clearly visible).

![image](https://user-images.githubusercontent.com/67039214/116566330-4a930d00-a924-11eb-8e36-899f3bf27ac0.png)

### Device information

* Device vendor / model and Android version: Redmi Note 8 Pro (Android 10)
* Firefox for Android version: Nightly 210331 17:00 (Build #2015801931) and Firefox Preview Version 1.0.2114

**Comment by neha-b2001** [comment]
I'm an Outreachy applicant, hence tagging @eliserichards.
If this is a valid issue, can I please be assigned to work on this?

**Comment by eliserichards** [comment]
I have tagged the UX team for feedback :) let's wait to see what they say before we start this issue.

**Comment by neha-b2001** [comment]
> I have tagged the UX team for feedback :) let's wait to see what they say before we start this issue.

Okay 👍

**Comment by topotropic** [comment]
Thanks! Let's use the following color for the chevron
` <color name="photonLightGrey05">#FBFBFE</color>`

**Comment by neha-b2001** [comment]
> Thanks! Let's use the following color for the chevron
> ` <color name="photonLightGrey05">#FBFBFE</color>`

Okay, thank you 👍 I'll make the changes right away

**Comment by lobontiumira** [comment]
Verified as fixed on the 6/9 Nightly build with Samsung Galaxy Note 8 (Android 9), and HTC 10 (Android 8).
Scanning the homescreen which has a collection (with dark theme enabled) does not give any suggestions regarding the drop-down button.
